.github/workflows/benchmark-runner.yml

```diff
@@ -152,7 +156,23 @@ jobs:
       profile_env: 1
       # TODO: change to "github-actions.profilers" when ready on grafana
       triggering_env: "circleci.profilers" # results generated with profilers attached are not mixed with the ones without it
-      module_path: bin/linux-x64-release-profile/search/redisearch.so
+      module_path: bin/linux-x64-release-profile/search-community/redisearch.so
       benchmark_glob: "vecsim*.yml"
       benchmark_runner_group_member_id: ${{ matrix.member_id }}
       benchmark_runner_group_total: ${{ strategy.job-total }}
+
+  benchmark-validation:
+    needs:
+      - benchmark-search-oss-standalone
+      - benchmark-vecsim-oss-standalone
+      - benchmark-search-oss-cluster-02-primaries
+      - benchmark-search-oss-cluster-04-primaries
+      - benchmark-search-oss-cluster-08-primaries
+      - benchmark-search-oss-cluster-16-primaries
+      - benchmark-search-oss-standalone-profiler
+      - benchmark-vecsim-oss-standalone-profiler
+    runs-on: ${{ vars.RUNS_ON }}
+    if: ${{ !cancelled() }}
+    steps:
+      - if: contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')
+        run: exit 1
```

Comment:
Why do we need this job?

Reply:
I thought this was a workaround for failing the flow if the matrix failed (as you did in merge queue). If not necessary I'll remove

Reply:
It is just a way to summarise all the jobs in a workflow into a single job with a uniform name, so we can make it the required job to approve a PR/merge it to the queue (even though these workflows have different jobs to run).

The `fail-fast: false` will eliminate the case of the workflow "cancelling" itself